src/Bonsai.ML.Visualizers/EllipseHelper.cs

```diff
@@ -0,0 +1,43 @@
+using MathNet.Numerics.LinearAlgebra;
+using System;
+
+namespace Bonsai.ML.Visualizers
+{
+    public static class EllipseHelper
```

Comment:
Build action is complaining about missing XML comment here, perhaps this was an internal class before and is now public? If so, it probably should have a summary string.

Reply:
Fixed